src/pydata_sphinx_theme/assets/styles/variables/_layout.scss

```diff
@@ -3,4 +3,10 @@
   * Overall Layout Variables
   */
   --pst-header-height: 60px;
+  --pst-header-article-height: calc(var(--pst-header-height) * 2 / 3);
```

Comment:
Can you provide a rationale in a comment for why this equation is used / what it's meant for?

Reply:
I just wanted to have 66% of the header height i.e. 2/3. If you think it's easier to understand, I can replace it by 40px